### PR TITLE
Absolute pathing in getrev.dm

### PR DESCRIPTION
**Pull request by Probe1**
SHOULD THIS FILE EVEN EXIST?

### CODE REVIEW

**Comment by Probe1** [comment]
I don't even know whats going on in this file or why we're checking for svn  revisions.

**Comment by Intigracy** [comment]
It gets the current github revision


 client/verb/showrevinfo()

**Comment by Intigracy** [comment]
Those last two should NOT be changed, they're not partial pathed they're global procs.

File: code/datums/helper_datums/getrev.dm [hunk 1]
```diff
@@ -13,88 +13,88 @@ var/global/datum/getrev/revdata = new("config/svndir.txt")
...
+	return {"<html>
+				<head>
+				</head>
+				<body>
+				<p><b>Server Revision:</b> [getRevisionText()]<br/>
+				<b>Author:</b> [commiter]</p>
+				</body>
+				<html>"}
 
-/proc/return_revision()
```
Comment (Intigracy) [inline code/datums/helper_datums/getrev.dm:90]: Reeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee

File: code/datums/helper_datums/getrev.dm [hunk 2]
```diff
@@ -13,88 +13,88 @@ var/global/datum/getrev/revdata = new("config/svndir.txt")
...
+/datum/getrev/proc/return_revision()
 	var/output =  "Sorry, the revision info is unavailable."
 	output = file2text(".git/refs/heads/Bleeding-Edge")
 	if(!output || output == "")
 		output = "Unable to load revision info from HEAD"
 	return output
 
-/client/verb/showrevinfo()
```
Comment (Intigracy) [inline code/datums/helper_datums/getrev.dm:97]: Reeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee

**Review by Intigracy — CHANGES_REQUESTED** [review]
These weren't partially pathed, they're global procs.

File: code/datums/helper_datums/getrev.dm (hunk 1 above)
Comment (Probe1) [reply code/datums/helper_datums/getrev.dm:90]: Just edit it faggioli

**Comment by Probe1** [comment]
Whys it call svn tho

**Comment by Probe1** [comment]
NEVERMIND THIS FILE.

**Pull request closed without merging**